src/image.tsx

```diff
@@ -228,6 +228,10 @@ export const ImageContent = React.forwardRef<HTMLImageElement, ImageProps>(
                         onError?.();
                     }}
                     onLoad={onLoadHandler}
+                    // Sometimes Safari doesn't render images completely
+                    // https://stackoverflow.com/questions/58323768/ios-safari-images-not-rendering-fully-cut-off
+                    // https://www.tunetheweb.com/blog/what-does-the-image-decoding-attribute-actually-do/
+                    decoding="async"
```

Comment:
async decoding seems to be the recommendation (by Jake Archibald) and the default in nextjs or wordpress